test/test.js

```diff
@@ -46,6 +46,23 @@ it('should skip @keyframes selectors', function () {
   assert.equal(out, expected)
 })
 
+it('should support an additional callback for prefix transformation', function () {
+  var out = postcss().use(prefix({
+    prefix: '.hello',
+    transform: function (prefix, selector, prefixAndSelector) {
+      if (selector === 'body') {
+        return 'body.body-picker'
```

Comment:
It's minor thing but I'd rather do `return 'body' + prefix` here, just so we're checking that the `prefix` is being passed correctly to this method.